speakers/riccardo-terrell/AcctorModelFSandAkka/src/AkkaActorModel

```diff
@@ -1 +0,0 @@
-Subproject commit 17677c6fd7d37019d3d9b4028485896227d2f1a1
```

Comment:
I have literally no idea what this means, or why it changed. I did not change it. It looks like changing this would be bad.

Maybe I need to upgrade my version of hg-git.